test/e2e/nodemaintenance_test.go

```diff
@@ -30,6 +32,26 @@ var (
 	podLabel             = map[string]string{"test": "drain"}
 )
 
+const (
+	scriptGetStatus = `OC="./cluster/kubectl.sh"; pwd; stat "$OC"; date; $OC get pods -n node-maintenance-operator; POD_NAME=$($OC get pods -n node-maintenance-operator  | grep node-maintenance-operator | awk '{print $1}'); echo "pod name: ${POD_NAME}"; $OC describe pod -n node-maintenance-operator ${POD_NAME}; $OC logs -n node-maintenance-operator ${POD_NAME} -c node-maintenance-operator`
+)
+
+func showDeploymentStatusScript(t *testing.T) {
+	mydir, _ := os.Getwd()
+	t.Logf("directory: %s", mydir)
+
+	clicmd := exec.Command("bash", "-xc", scriptGetStatus)
+	output, err := clicmd.CombinedOutput()
+	if err != nil {
+		t.Logf("can't get cluster status: %v\n", err)
+	}
+
+	if output != nil {
+		soutput := string(output)
+		t.Logf("cluster status: %s", soutput)
+	}
+}
+
```

Comment:
Please remove. We'll pick that up from https://github.com/kubevirt/node-maintenance-operator/pull/69

Reply:
done.